src/components/App/App.jsx

```diff
@@ -39,17 +39,13 @@ function App() {
           <Route exact path="/search">
             <SearchPage />
           </Route>
-          {/* For protected routes, the view could show one of several things on the same route.
-            Visiting localhost:3000/user will show the UserPage if the user is logged in.
-            If the user is not logged in, the ProtectedRoute will show the LoginPage (component).
-            Even though it seems like they are different pages, the user is always on localhost:3000/user */}
-          <ProtectedRoute
-            // logged in shows UserPage else shows LoginPage
+
+          <Route
             exact
             path="/admin"
           >
             <AdminPage />
-          </ProtectedRoute>
+          </Route>
```

Comment:
Should this remain Protected?  Anyone not logged in would be pushed to a 403 error.

Reply:
It should be protected in the final product but we don't yet have the admin functionality in - we will make sure to cover this when we are re-protecting routes and doing conditional rendering.